### PR TITLE
CPMap Docs and 5.4.0 Release Notes

### PR DESCRIPTION
**Pull request by yuce**

### CODE REVIEW

File: docs/modules/ROOT/pages/clc-cpmap.adoc [hunk 1]
```diff
@@ -0,0 +1,369 @@
...
+
+[source,bash]
+----
+clc cpmap -n my-cpmap set -k string hello -v f32 19.94
+----
+
+
+== clc cpmap put
+
+Put sthe given value in the given CPMap and return sthe old value.
```
Comment (rebekah-lawrence) [inline docs/modules/ROOT/pages/clc-cpmap.adoc:184]:
```suggestion
Puts the given value in the given CPMap and returns the old value.
```

**Review by rebekah-lawrence — APPROVED** [review]
A typo only to correct; otherwise LGTM so approving in advance.

File: docs/modules/ROOT/pages/clc-cpmap.adoc (hunk 1 above)
Comment (yuce) [reply docs/modules/ROOT/pages/clc-cpmap.adoc]: Thanks Rebekah,
I've fixed the typo and added the reference for CPMap advanced script functions at:
https://github.com/hazelcast/clc-docs/pull/19/commits/298d2a3c821552757a99b36086dac604e0f36b43

Comment (rebekah-lawrence) [reply docs/modules/ROOT/pages/clc-cpmap.adoc]: All changes look good. Thanks Yuce

**Pull request merged**